devicon.json

```diff
@@ -3109,10 +3109,12 @@
         "versions": {
             "svg": [
                 "original",
-                "plain"
+                "plain",
+                "line"
             ],
             "font": [
-                "plain"
+                "plain",
+                "line"
             ]
         },
         "color": "#ffcfab",
```

Comment:
Shouldn't we add aliases? The other versions have text on it, so we can create a `wordmark` alias.

Reply:
Agreed